src/editor.js

```diff
@@ -264,9 +275,77 @@ const ContentPanel = ({
         codeMirrorElement is a promise resolved by codemirror(), hence the
         until directive. Once resolved, content can be empty.
       -->
+    ${TemplateLightbox({isTemplateLightboxDisplayed})}
     ${until(codeMirrorElement || Textarea({content}))}
   </div>
 `;
+const dispatchLoadHtmlFile = redispatchAs(g('load-html'));
+
+const TemplateLightbox = ({isTemplateLightboxDisplayed}) => html`
+  <div
+    class="${n('template-lightbox')}"
+    ?hidden=${!isTemplateLightboxDisplayed}
+  >
+    ${getNamesOfTemplates()}
+  </div>
+`;
+
+function getNamesOfTemplates() {
+  return html`
+    <div class="${g('flex-center')}">
+      <div
+        class="${n('template')}"
+        @click=${dispatchLoadHtmlFile}
+        data-name="app-install-ads"
+      >
+        <img width="100%" src="/static/app_install.jpg" />
+      </div>
+      <div
+        class="${n('template')}"
+        @click=${dispatchLoadHtmlFile}
+        data-name="multi-image-ads"
+      >
+        <video
```

Comment:
Put each of these files under their template folder, named `_preview.ext` (`ext` is their current extension, `mp4`, `jpg`, whatever). The snippet above will handle them specially based on this name so that you don't have to hardcode them here.